content/documentation/github.md

```diff
@@ -1,101 +1,143 @@
 ---
-title: GitHub for Quire
-weight: 600
+title: Project Management with GitHub
+weight: 4150
 type: essay
 ---
 
-GitHub is a project management platform centered around git-based version control. That means it’s a good place to host your project and track changes to it, whether you’re working on your own, or with a team of collaborators. The Getty uses GitHub to manage all of its publications, as well as Quire itself.
+[GitHub](https://github.com) is a project management platform centered around git-based version control. While it was originally developed as a useful tool for developers, it is now used by artists, writers, and other non-tech professionals. In fact, Getty uses GitHub to manage all of its publications, as well as Quire itself. We strongly recommend using GitHub as a tool for managing your Quire projects (whether collaborating or working solo).
 
-To get started, you just need to [sign up for a free GitHub account](https://github.com/join), and [download and install the free GitHub Desktop software](https://desktop.github.com/). (Advance users will often use git and GitHub from their {{< q-def "command-line shell" >}}.)
+**Here are some of GitHub's main advantages:**
 
-## Hosting Your Project Code on GitHub
+- GitHub is free to use.
+- It's a secure platform for hosting your project's code and creating an electronic back-up of your work.
+- GitHub saves the entire version history of your project (including deleted files) enabling you to track changes, revert changes, and restore earlier versions.
+- You can use GitHub to create test versions of your project and experiment with new ideas before officially incorporating them.
+- Github makes working with collaborators easy. You and your colleagues can work in different versions of the project, review and comment on one another's work, and consolidate any approved changes into the master project.
```

Comment:
Do we need to mention that it's not like track changes in Word or Google Docs? Temper expectations a little? not sure

Reply:
I don't think we need to include that here, because aren't suggested a parallel to Word or Google Docs anywhere. 

I think maybe it's something we mention in our trainings to clear up any past confusion created by that phrasing.